server/ee.js

```diff
@@ -0,0 +1,35 @@
+const express = require('express');
+const ejs = require('ejs');
+const bp = require('body-parser');
+
+const app = express();
+
+app.use(express.static('public'));
+app.use(bp.json());
+app.engine('html', ejs.renderFile);
+app.get('/:id', (req, res) => {
+  res.render('../public/index.html');
+});
+app.set('view engine', 'html');
+
+const port = process.env.PORT || 4000;
+app.listen(port, () => { console.log(`Listening on port ${port}`); });
+
+
+<!DOCTYPE html>
+<html>
+<head>
+  <title>Calendar-Proxy</title>
+</head>
+<body>
+  <div id="app">NO REACT</div>
+  <script crossorigin src="https://unpkg.com/react@16/umd/react.development.js"></script>
```

Comment:
since we are going directly to the bundle.js (no react), you probably don't need the react cdns.